packages/kbn-test/src/kbn_client/kbn_client_import_export.ts

```diff
@@ -66,6 +68,19 @@ export class KbnClientImportExport {
     const formData = new FormData();
     formData.append('file', objects.map((obj) => JSON.stringify(obj)).join('\n'), 'import.ndjson');
 
+    if (options?.space) {
+      this.log.info('creating space', options.space);
+
+      try {
+        await this.spaces.create({
+          id: options.space,
+          name: options.space,
+        });
+      } catch (e) {
+        this.log.warning('creating space failed:', e.message);
```

Comment:
I assume the intention here is to ignore "space already exists" errors, but I think we should check for those specifically and throw all other types of errors.

Reply:
Yeah, I agree. Will update.

Reply:
0eb68eb12b67519d9360e9dd82126c27bff7ca41